extensions/amp-story/0.1/media-pool.js

```diff
@@ -45,6 +48,62 @@ export const MediaType = {
 };
 
 
+/** @enum {string} */
+export const MediaPoolEvents = {
+  ALLOCATED: 'amp:pool:allocated',
+  DEALLOCATED: 'amp:pool:deallocated',
+};
+
+
+/**
+ * Playback events to bubble.
+ * By default, these don't bubble. We need to re-dispatch so that components
+ * can listen to these events without access to the underlying media element.
+ * @private @const {!Array<string>}
+ */
+const NON_BUBBLING_EVENTS = [
```

Comment:
nit: I think the name `NON_BUBBLING_EVENTS` is weird here; these are events that don't bubble by default, but we will bubble, so at the end of the day, they still bubble...

Maybe `EVENTS_TO_PROPAGATE` or `EVENTS_TO_BUBBLE`?

Reply:
I'll be difficult and will use `EVENTS_TO_FORWARD` :)